### PR TITLE
auto OCP-38812

### PR DESCRIPTION
**Pull request by lihongan**
logs:
upgrade-prepare: https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/265560/console

upgrade-check: https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/265651/console

@quarterpin please help take a look. 

cc @jechen0648

### CODE REVIEW

**Comment by quarterpin** [comment]
/lgtm 

Thanks, @lihongan .

File: features/upgrade/routing/upgrade.feature [hunk 1]
```diff
@@ -89,3 +89,91 @@ Feature: Routing and DNS related scenarios
     Then the expression should be true> deployment("router-default").current_replicas(cached: false) == <%= cb.num_workers %>
     And <%= cb.num_workers %> pods become ready with labels:
       | ingresscontroller.operator.openshift.io/deployment-ingresscontroller=default |
+
+
+  # @author hongli@redhat.com
+  # @case_id OCP-38812
```
Comment (liangxia) [inline features/upgrade/routing/upgrade.feature:95]: I think we do not use case ID for prepare.

Comment (lihongan) [reply features/upgrade/routing/upgrade.feature]: Thank you. Yes, just keep the case ID for upgrade-check.

**Pull request merged**